Bearsampp/Bearsampp
# Change icon in restart

This changes the icon for restart box to our Bearsampp icon.
To test:
compile any build and start it.
when you get the popup to restart, check the icon in the toolbar.
related to https://github.com/Bearsampp/.teams/issues/14
